### ISSUE TITLE
worker/server issue

### ISSUE THREAD

**Issue by jonnymangothinkjam** [AbdulBsit/action-watermark#2]
I'm trying to run this through the worker/server on a remote AWS set up of nexrender and am getting the following error:

 error occurred: NestedError: Could not require module 'action-watermark'
    at requireg (C:\snapshot\nexrender\packages\nexrender-worker\node_modules\@nexrender\core\node_modules\requireg\lib\requireg.js:14:11)
    at C:\snapshot\nexrender\packages\nexrender-worker\node_modules\@nexrender\core\src\tasks\actions.js
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async start (C:\snapshot\nexrender\packages\nexrender-worker\src\index.js)
Caused By: C:\Users\Administrator\AppData\Roaming\npm\node_modules\action-watermark\index.js:76
  let input = options.input || job.output;
      ^^^^^

everything is installed 

project json

```
actions: {
            postrender: [
                {
                module: '@nexrender/action-encode',
                preset: 'mp4',
                output: 'encoded.mp4'
            },
            {
                module: "action-watermark",
                input: 'encoded.mp4',
                watermark: "https://....link_to_watermark.png",
                output: 'file_watermarked.mp4',
            },
            {
                module: "@nexrender/action-upload",
                input: "encoded.mp4",
                provider: "s3",
                params: {
                    "region": "eu-west-1",
                    "bucket": "XXXX",
                    "key": 'file.mp4',
                    "acl": "private"
                }
            }
```

**Comment by AbdulBsit** [comment]
Hello, Sorry for the late reply, did you imported action-watermark module in your file?

**Comment by AbdulBsit** [comment]
resolved in #3
